AndriiDryha/HW11/PRACTICAL TASKS [LESSON 11].py

```diff
@@ -0,0 +1,52 @@
+# # Task 1
+
```

Comment:
good job
but look at this version of your code
def number_type():
    try:
        age = int(input('Enter your age:\n'))
        if age > 0:
            if age % 2 == 0:
                print(f"Age {age} is even.")
            else:
                print(f"Age {age} is odd.")
        else:
            print("Age should be above 0")
            number_type()
    except ValueError:
        print("Invalid input. Age should be a positive integer.")
        number_type()
    except KeyboardInterrupt:
        print("\nExiting...")
    except Exception as e:
        print(f"An error occurred: {e}")
        number_type()

number_type()